app/Http/Controllers/Cms/EventController.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace App\Http\Controllers\Cms;
+
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+
+use App\Event;
+use Carbon\Carbon;
+
+class EventController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $events = Event::all();
+
+        return view('cms.event.index', [
+            'events' => $events
+        ]);
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return view('cms.event.create');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $validatedData = $request -> validate([
+            'name' => 'required|max:255',
+            'description' => 'required',
+            'place' => 'required|max:255',
+            'course' => 'required|max:255',
+            'startDate' => 'required|date',
+            'duration' => 'required',
+            'enrollable' => 'boolean',
+        ]);
+
+
+        if (! array_key_exists('enrollable', $validatedData)) {
+            $validatedData['enrollable'] = "0";
+        }
+
```

Comment:
Não consegui encontrar outra forma de validar checkbox, se alguem tiver alguma ideia agradeço.

Reply:
Eu achei essa [pergunta](https://stackoverflow.com/questions/2268887/how-do-i-see-which-checkbox-is-checked) no stackoverflow que o cara sugere usar o `isset()`, dai ficaria 
```php
isset($validatedData['enrollable'])
```
 mas tu que sabe, no fim das contas o resultado é o mesmo